boards/arm/96b_carbon/Kconfig.defconfig

```diff
@@ -10,14 +10,4 @@ if BOARD_96B_CARBON
 config BOARD
 	default 96b_carbon
 
-if I2C_1
-
-config I2C_1_DEFAULT_CFG
```

Comment:
The default value for all I2C_n_DEFAULT_CFG is 0x00. Why should we change it the board defconfig? Users should configure it in their applications.